sopel/modules/wikipedia.py

```diff
@@ -327,7 +328,13 @@ def mw_image_description(server, image):
 def mw_info(bot, trigger, match=None):
     """Retrieves and outputs a snippet of the linked page."""
     server = match.group(1)
-    query = unquote(match.group(2))
+
+    # NOTE: Discard the query string if there is one, avoids the error in gh-2412 https://github.com/sopel-irc/sopel/issues/2412
+    # TODO: replace this usage when/if gh-2413 is resolved https://github.com/sopel-irc/sopel/issues/2413
```

Comment:
If you put a direct number reference, it's linked in GH code view, btw.

```suggestion
    # NOTE: Discard the query string if there is one; see #2412
    # TODO: replace this usage when/if #2413 is resolved
```

(I also don't know if this TODO is useful, considering how uncertain it is that anything will be implemented based on #2413; and if it is, we'll certainly take a pass through the plugins that are still in core to find uses of `urlparse` or any straggling regex-based "parsing" of links that might still be left by then.)

Reply:
Hmm, fair enough, I'll drop it